CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+##2020-11-03 - Release 2.0.0
+###Summary
+
+Major version change -
```

Comment:
maybe just provide the URL for the changes:
https://www.elastic.co/guide/en/beats/libbeat/7.9/breaking-changes-7.0.html#breaking-changes-7.0